adyen.php

```diff
@@ -684,6 +724,42 @@ public function hookPayment()
             }
         }
 
+        if (!empty($paymentMethods['paymentMethods'])) {
```

Comment:
Does it worth to follow WET and not creating a helper function which prepares the issuerlist ?

Reply:
What's WET?